target/snitch_cluster/sw/Makefile

```diff
@@ -30,5 +32,5 @@ apps: $(RUNTIME) math $(SNAX_MAC) $(SNAX_GEMM)
 tests: $(RUNTIME) math $(SNAX_MAC) $(SNAX_GEMM)
 	$(MAKE) -C $@ $(TARGET)
 
-runtime/rtl runtime/banshee math $(SNAX_MAC) $(SNAX_GEMM):
+runtime/rtl runtime/banshee runtime/rtl-generic math $(SNAX_MAC) $(SNAX_GEMM):
```

Comment:
Are there dependencies between `runtime/rtl` and `runtime/rtl-generic` ?

Maybe we can just do the same `$(RUNTIME)` for this part.

Reply:
@rgantonio, no there are none. Actually now that I'm thinking of this, I would actually prefer it if we could change the other targets on this line also to be `snax/mac` and `snax/gemm` instead, in that way it is nice and explicit what is being built, but if the name changes then you get a lot of place to change it....
I don't know what the best solution is hahah. Let me know if you still like me to change it

Reply:
@JosseVanDelm let's change it to the `snax/mac` and the `snax/gemm` version. In the latest Snitch Cluster they used it this way :)

Reply:
@JosseVanDelm maybe change the `$(SNAX_MAC)` and `$(SNAX_GEMM)` as well? So it's uniform? Just an idea haha although it's just a minor thing...